Sources/Widgets/Representations/ResliceCursorContextRepresentation/index.js

```diff
@@ -72,6 +72,12 @@ function vtkResliceCursorContextRepresentation(publicAPI, model) {
   model.pipelines.axes.push(axis1);
   model.pipelines.axes.push(axis2);
 
+  model.rotationActorsList = [];
+  model.rotationActorsList.push(model.pipelines.axes[0].rotation1.actor);
+  model.rotationActorsList.push(model.pipelines.axes[0].rotation2.actor);
+  model.rotationActorsList.push(model.pipelines.axes[1].rotation1.actor);
+  model.rotationActorsList.push(model.pipelines.axes[1].rotation2.actor);
```

Comment:
`rotationActorList` or `rotationActors`, I personally prefer `rotationActors` (the `List` suffix does not bring much information)